doc/references.bib

```diff
@@ -47,6 +47,16 @@ @article{AcunzoEtAl2012
   year = {2012}
 }
 
+@article{Alday2019,
+  title = {How much baseline correction do we need in ERP research? Extended GLM model can replace baseline correction while lifting its limits},
```

Comment:
```suggestion
  title = {How much baseline correction do we need in {ERP} research? Extended {GLM} model can replace baseline correction while lifting its limits},
```

This will force uppercase for the acronyms